guides/examples/basic-web-app.md

```diff
@@ -164,11 +260,38 @@ ockam policy create --at web --resource tcp-inlet --expression '(= subject.compo
 ockam tcp-inlet create --at /node/web --from 127.0.0.1:5432 --to /project/default/service/forward_to_db/secure/api/service/outlet
 ```
 
-Take note of the `--from` and `--to` values above. The `--from` is telling the node to listen on port 5432, the default postgres port, and to forward it `--to` the forwarder service to the database that we created in the previous section. This means requests to localhost:5432 will be forwarded to whatever node has registered as `db`, wherever it is!
+Take note of the `--from` and `--to` values above. The `--from` is telling the node to
+listen on port `5432`, the default postgres port, and to forward it `--to` the forwarder
+service to the database that we created in the previous section. This means requests to
+`localhost:5432` will be forwarded to whatever node has registered as `db`, wherever it is!
+
+Which means if you start your web app the counter will continue incrementing just as it
+did before, with zero code changes to your application
+
+You could also extend this example by moving the Postgres service into a Docker container
+or to an entirely different machine. Once the nodes are registered the demo will continue
+to work, with no application code changes and no need to expose the Postgres ports
+directly to the internet.
+
+Here's a diagram w/ some architecture details of what we have done in this exercise.
+
+<figure><img src="../../.gitbook/assets/infrastructure.webp" alt=""><figcaption></figcaption></figure>
```

Comment:
I am not convinced that this diagram really brings anything at this stage. The purpose of this diagram is:

 - to show that there are 3 "entities" involved in exposing a service, with a relay in the middle
 - but mostly to show that messages go through several layers of protocols: TCP, Ockam routing, Ockam secure channels

And I think that this second point is mostly a concern for the documents on routing and secure channels.

Reply:
done